dbschema/variance-explanation.esdl

```diff
@@ -0,0 +1,16 @@
+module ProgressReport {
+  type VarianceExplanation {
+    required report: default::ProgressReport {
+      readonly := true;
+    };
+    
+    reasons: array<str> {
+      readonly := true;
+    };
```

Comment:
```suggestion
    multi reasons: str;
```
Editable. And I think we should prefer `multi` over `array`.

Reply:
Ah, I didn't know `multi` applied to things other than links.  Gtk.